packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_counter/views/counter_page.dart

```diff
@@ -62,26 +60,33 @@ class CounterPage extends StatelessWidget implements AutoRouteWrapper {
         : Container(),
   );
 
+  Widget _buildLoadingScreen() =>
+      const Center(child: CircularProgressIndicator());
 
   Widget _buildErrorScreen(
       BuildContext context,
+      String errorMessage,
       CounterBlocType bloc,
       ) =>
-      Container(
-        child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-          Text(
-              'Your API is not running. \nRun command \'bin/start_server.sh\' and try again.',
-              textAlign: TextAlign.center,
-              style: context.designSystem.typography.headline5,
-          ),
-          ElevatedButton(
-              onPressed: bloc.events.current,
-              child: Text(
-                'RETRY',
-                style: context.designSystem.typography.buttonMain,
-              )),
-        ]),
-      );
+      !errorMessage.contains('Http status error [422]')
+          ? Container(
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                'Your API is not running. \nRun command \'bin/start_server.sh\' and try again.',
+                textAlign: TextAlign.center,
+                style: context.designSystem.typography.headline5,
+              ),
+              ElevatedButton(
+                  onPressed: bloc.events.current,
+                  child: Text(
+                    'RETRY',
+                    style: context.designSystem.typography.buttonMain,
+                  )),
+            ]),
+      )
+          : _buildCount();
```

Comment:
Why do you display the count on `_buildErrorScreen` ? It should be used to present error-related UI states.

Reply:
Because I can not filter errors that triggers buildError, so I have to make a check in the UI to not displaying error screen, when the Exception is for showing snack bar (when we reach end value).

Reply:
But why are you even displaying a snack bar in the first place? Shouldn't the error screen display a message that the server is not running?